examples/js/effects/VREffect.js

```diff
@@ -42,8 +42,26 @@ THREE.VREffect = function ( renderer, onError ) {
 
 			if ( onError ) onError( 'HMD not available' );
 
+			return;
+
 		}
 
+		if ( !deprecatedAPI ) return;
```

Comment:
yeah, I like that style. but I didn't see that usage elsewhere. mr.doob style rulez.